content/scout/data-handling.md

```diff
@@ -44,6 +53,11 @@ information on the SBOM. If there's a match, the results of the match are
 displayed in the user interfaces where Docker Scout data is surfaced, such as
 the Docker Scout Dashboard and in Docker Desktop.
 
```

Comment:
I wonder if this paragraph should come before the list?

Reply:
yeah that makes sense!